src/app/public/modules/popover/popover.directive.ts

```diff
@@ -179,36 +179,42 @@ export class SkyPopoverDirective implements OnInit, OnDestroy {
       .fromEvent(element, 'click')
       .takeUntil(this.ngUnsubscribe)
       .subscribe(() => {
-        this.togglePopover();
+        if (this.skyPopover) {
```

Comment:
👇 Just adding a check to make sure the `skyPopover` exists before running any of the event handlers.